### PR TITLE
Adding Metrics example

### PR DESCRIPTION
**Pull request by pallavipannu**

### CODE REVIEW

File: metrics/example/metrics.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import json
+import math,sys,time
+import random
+import requests
+import datetime
+
+print("*******************Custom Metrics Generator***************************")
+url = "http://dkube-exporter.dkube:9401/modelmonitor/metrics"
+
+monitor_id = sys.argv[1]
```
Comment (songole) [inline metrics/example/metrics.py]: Isn't this available as ENV or in config.json?

File: metrics/example/metrics.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+print("monitor id is ",monitor_id)
+
+metric_value = random.random()
+
+payload = json.dumps({
+  "id": monitor_id,
+  "timestamp": math.ceil(datetime.datetime.utcnow().timestamp()),
```
Comment (songole) [inline metrics/example/metrics.py]: config.json should be mounted here. Can you provide a function to calculate the end of last sample based on run_frequency. You can use metrics.py in monitoring repo for reference.

Comment (songole) [reply metrics/example/metrics.py]: @gaurav-oc make sure config.json is mounted for custom image

File: metrics/example/metrics.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+"metrics": [
+    {        "label_values": {"run_id": "c07a61a4-f660-4a7f-84e1-05f7330e748f","mm_feature":"age"},
```
Comment (songole) [inline metrics/example/metrics.py]: I am not sure if run_id is still needed. We could go by timestamp.

File: metrics/example/metrics.py (hunk 2 above)
Comment (gaurav-oc) [reply metrics/example/metrics.py]: inside custom container ?

Comment (gaurav-oc) [reply metrics/example/metrics.py]: its mounted same as we mount for other workflow container

**Pull request merged**